blog-posts/06_03_2021.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Home</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <link rel="stylesheet" href="styles/style.css">
```

Comment:
Just needed to add the `..` be able to access the `styles` folder. 

```suggestion
    <link rel="stylesheet" href="../styles/style.css">
```